(dp1
S'output'
p2
S"<type 'exceptions.AttributeError'> 'Table' object has no attribute 'vcod_presenca'"
p3
sS'layer'
p4
S'/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x7f73a5c9b730, file "/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py", line 10>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/leonardo/Downloads/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py", line 199, in <module>\n  File "/home/leonardo/Downloads/web2py/gluon/globals.py", line 419, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py", line 176, in busca_aluno\n    m = db(db.frequencia.vcod_presenca == cod_presenca).select(\nAttributeError: \'Table\' object has no attribute \'vcod_presenca\'\n'
p11
s.